package/pom.xml

```diff
@@ -12,10 +12,6 @@
   <name>10 b benchmark package - 10 b benchmark package</name>
   <version>11.2.35-SNAPSHOT</version>
```

Comment:
Why the version override?

Reply:
We need to have a unique version when deploying the package, thus the different version numbers.